packages/context/src/app/AppProvider.tsx

```diff
@@ -2,7 +2,7 @@
 // This file is licensed under the MIT License.
 
 import { createContext, ReactNode, useCallback, useContext } from 'react';
-import { useApiServices } from '../api';
+import { useApiServices } from '../api/index.js';
 import { Obj } from '../objects';
```

Comment:
This also needs updating.  Probably need to go through the entire project, `module` packages can't import from a directory, you have to specify the file including the extension.

Reply:
Good catch. I went through every file in the context and double checked that the extensions were included.